### PR TITLE
Add containerd implementation

### PR DESCRIPTION
**Pull request by nathana1**

### CODE REVIEW

File: docs/implementations/containerd.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Containerd
+Containerd is a container runtime.  This document describes how it authenticates to remote registries.
+## Authentication
+- Authentication information is stored on a per host basis.
+    - `service` and `realm` are assumed to be constant per host.
+    - Individual `scope` values are added without any sort of merging (i.e. stored as both `repository:<repo>:pull` and `repository:<repo>:pull,push` instead of just `repository:<repo>:pull,push`).
+    - This also applies to hosts that are the target of redirection.
+- Requests are retried up to 5 times. Only status codes 401 (Unauthorized), 405 (Method Not Allowed), 408 (Request Timeout), and 429 (Too Many Requests) are retried.
+- Multiple scopes are included in push or repository remounts
```
Comment (sudo-bmitch) [inline docs/implementations/containerd.md:9]: Are these scopes generated by containerd, read from the WWW-Authenticate header, or both?

**Review by sudo-bmitch — APPROVED** [review]
LGTM, just one non-blocking question.

**Comment by sudo-bmitch** [comment]
I spoke too soon. The linter needs fixes. You can run that locally with `make lint-md`.

File: docs/implementations/containerd.md (hunk 1 above)
Comment (nathana1) [reply docs/implementations/containerd.md:12]: They all come from the www-authenticate header.

**Review by sudo-bmitch — APPROVED** [review]

**Pull request merged**